test/core-tests/test-files.js

```diff
@@ -26,9 +27,10 @@ describe('files', () => {
     arr.push(filePair)
     ipfs.files.add(arr, (err, res) => {
       expect(err).to.not.exist
-      expect(res[0].path).to.equal('data.txt')
```

Comment:
The path check should now be covered by <a href="https://github.com/ipfs/interface-ipfs-core/pull/20/commits/8c5eeb5591aa2378301911fe9f3a965322b529ca#diff-6581b67864c5dbbaa734e512057741ceR53">this</a> once interface-core spec is finalized